Library/Formula/yelp-tools.rb

```diff
@@ -0,0 +1,19 @@
+class YelpTools < Formula
+  homepage "https://github.com/GNOME/yelp-tools"
```

Comment:
Could you please add a short description of the formula, using `desc "…"`? Thanks!